### ISSUE TITLE
Tampermonkey Script?

### ISSUE THREAD

**Issue by MintMain21** [einaregilsson/Redirector#332]
I have a custom Add-On collection for Firefox Android that allows me to install and use Redirector on Android. This requires I use either Firefox Beta, Firefox Nightly, or a fork like Fennec F-Droid. 

Recently Mozilla announced that a new set of extensions would be available in Nightly's default collection (https://blog.mozilla.org/addons/2022/12/15/new-extensions-available-now-on-firefox-for-android-nightly) and I assume the plan is to port these into Beta and Stable as well. While Redirector is not among these extensions, TamperMonkey is.

I've already experimented with a few TamperMonkey scripts, and while I've found some decent redirection scripts, I still haven't found one that presents the functionality of Redirector in a user friendly way.

TLDR, all I want is a simple script that can recreate the functionality of creating a Redirector Rule with a clear exception. 

To be more specific, I want to redirect Invidious pages with appropriate URL parameters without redirecting pages already featuring those parameters. 

I'll create a separate issue regarding how we can push Redirector into official Android Support based on the latest announcements.

**Comment by MintMain21** [comment]
Nevermind, I found this script (https://greasyfork.org/en/scripts/452774-genius-instant-old) and just edited it to work for the Invidious Instances I want and the URL Parameters I want.

Maybe there's a way to make it more complex so that it can, for example, redirect the homepage with different parameters than video pages, but this is good enough for me. For now at least.

I'll leave the issue open if anyone wants to contribute anything, otherwise all is well.

**Comment by MintMain21** [comment]
I have an official repository for my own URL Parameter userscript now. If anyone wants to contribute to that feel free to.

https://github.com/MintMain21/Invidious-URL-Parameters-Userscript

**Comment by Gitoffthelawn** [comment]
@MintMain21 Looks good.  Thanks for posting it.
